mix.exs

```diff
@@ -42,7 +42,8 @@ defmodule Hexdocs.MixProject do
   defp releases() do
     [
       hexdocs: [
-        include_executables_for: [:unix]
+        include_executables_for: [:unix],
+        reboot_system_after_config: true
```

Comment:
This preserves config/releases.exs behavior: https://hexdocs.pm/mix/Mix.Tasks.Release.html#module-config-providers